.vscode/launch.json

```diff
@@ -11,6 +11,23 @@
 			"debugAdapter": "dlv-dap",
 			"mode": "remote",
 			"port": 2345
-		}
+		},
+        {
+            "name": "Test Current File",
+            "type": "go",
+            "request": "launch",
+            "mode": "test",
+            "program": "${relativeFileDirname}",
```

Comment:
Can't use file since it needs the package it is testing defined in its directory. As a result, if multiple test files are in the same directory, they will all get run, but "Test Current Directory" doesn't have the same ring to it